technical-reference/reference-technical-articles/technical-doc-xvs-bridge.md

```diff
@@ -0,0 +1,221 @@
+# Cross-Chain Bridge Documentation
+
+This documentation provides detailed instructions and explanations for using the Cross-Chain Bridge. The Bridge allows users to transfer tokens between different blockchain networks, specifically between the BNB and the Destination Chain. The system consists of multiple contracts, including XVSBridgeAdmin, XVSProxySrc, XVSProxyDest, and XVS token contracts.
+
+**_The functionality of the protocol relies on LayerZero for the seamless transfer of XVS tokens across different networks. Consequently, the security and integrity of the token on each network are subject to potential vulnerabilities inherent in the bridging mechanism. It is essential to note that these risks are a general characteristic of integrating with network bridges and do not stem from any particular weaknesses within the token implementation._**
+
+## Table of Contents
+
+1. [Getting Started](#1-getting-started)
+   - [1.1. Approving XVS Tokens](#11-approving-xvs-tokens)
+   - [1.2. Estimating Transaction Fees](#12-estimating-transaction-fees)
+2. [Transferring Tokens](#2-transferring-tokens)
+   - [2.1. Sending Tokens](#21-sending-tokens)
+3. [Receiving Tokens on the Destination Chain](#3-receiving-tokens-on-the-remote-chain)
+4. [Transferring Tokens Back to the BNB](#4-transferring-tokens-back-to-the-bnb)
+5. [Monitoring Transaction Status](#5-monitoring-transaction-status)
+6. [Security and Risks](#6-security-and-risks)
+
+   - [6.1. Ownership Transfer](#61-ownership-transfer)
+   - [6.2. Pause and Resume](#62-pause-and-resume)
+   - [6.3. Limit the Amount of XVS Transfers](#63-limit-the-amount-of-xvs-transfers)
+   - [6.4. Transfer Delays](#64-transfer-delays)
+   - [6.5. Token Controller Contract](#65-token-controller-contract)
+   - [6.6. Cap on Token Minting](#66-cap-on-token-minting)
+
+7. [Contract Details](#7-contract-details)
+   - [7.1. XVSBridgeAdmin](#71-xvsbridgeadmin)
+   - [7.2. XVSProxySrc](#72-xvsproxysrc)
+   - [7.3. XVSProxyDest](#73-xvsproxydest)
+   - [7.4. XVS Token](#74-xvs-token)
+8. [Variables and Configuration](#8-variables-and-configuration)
+9. [Additional Features](#9-additional-features)
+   - [9.1. Oracle Integration](#91-oracle-integration)
+   - [9.2. Whitelist Mechanism](#92-whitelist-mechanism)
+   - [9.3. Transaction Limits](#93-transaction-limits)
+   - [9.4. Pause and Unpause Mechanism](#94-pause-and-unpause-mechanism)
+10. [Possible Failures of Bridging XVS Tokens](#10-possible-failures-of-bridging-xvs-tokens)
+    - [10.1. Sending XVS tokens from the source chain](#101-sending-xvs-tokens-from-the-source-chain)
+    - [10.2. Receiving XVS tokens on the destination chain](#102-receiving-xvs-tokens-on-the-destination-chain)
+
+## 1. Getting Started <a name="getting-started"></a>
```

Comment:
We don't need to include the `<a>` item in the headers. Gitbook will do it automatically